src/app/modules/attribution/attribution.html

```diff
@@ -23,7 +23,7 @@
       </div>
       <div class="col-md-12">
         <div class="content-group attribution-submit-button">
-          <ng-container *ngIf="!name">
```

Comment:
![surfing-be-gone](https://user-images.githubusercontent.com/15100429/46880155-e5cf4000-ce15-11e8-8f57-7ca4b992c5fc.gif)

I noticed that if I...

1. start on `/new`
2. add `?name=surfing_is_life`
2. go back to just `/new`

the nav is still missing...

perhaps the logic on ngInit can always set the form name = this.name? so when its there we hide, when its not line 26 will return false (since this.name is undefined, i think?)

Reply:
So you'll actually notice that I removed the `*ngIf="!name" and instead am using the navigation service to hide/show the navigation components. `MainComponent.ts` now calls `nav.show` and that is what is making the navigation bar show when you open the app from the homepage. Otherwise I assumed they were accessing it from outside the application (like an iFrame)

Reply:
maybe we should add a more specific parameter like "?navigation=false" though which will hide/show navigation elements